## Tom Gaunt

I lead the [‚ÄúData Mining Epidemiological Relationships‚Äù](http://www.biocompute.org.uk/) programme in the [MRC Integrative Epidemiology Unit](http://www.bristol.ac.uk/integrative-epidemiology/) at the University of Bristol. We are interested in understanding the mechanisms of disease, and approach this through the integration of diverse biomedical and epidemiological data and the development of software tools for analysis of these data. One of our key developments is [EpiGraphDB](http://epigraphdb.org/), a database and software platform that integrates epidemiological and biomedical data to support mechanism discovery and aid causal inference.

### Collaborations and opportunities

We are always interested in hearing from potential collaborators or from talented graduate or postgraduate researchers who wish to pursue a career in software engineering applied to bioinformatics, molecular epidemiology or data science. 

üì´ [Get in touch!](https://www.bris.ac.uk/contact/person/getDetails?personKey=f7rKQdp39cRsRoJTnrfZieha0e3t2j)

### Selected software projects

The following software platforms have been developed by members of my research group and collaborators. Most are hosted on the <a href="https://github.com/MRCIEU/">MRC IEU GitHub organisation <img src="http://i.imgur.com/9I6NRUm.png"></a>.

<ul>
  <li><a href="http://www.epigraphdb.org/">EpiGraphDB</a> integrates epidemiological and biomedical data to support mechanistic and causal inference. </li>
  <li><a href="https://gwas.mrcieu.ac.uk/">IEU OpenGWAS database</a> contains thousands of publicly available GWAS for download or use in MR-Base. </li> 
  <li><a href="http://vectology.mrcieu.ac.uk/">Vectology</a> uses sentence embedding methods to compare distances between biomedical terms.</li>
  <li><a href="http://www.mrbase.org/">MR-Base</a> enables online Mendelian randomization analysis using a comprehensive manually curated database of GWAS studies.</li>
  <li><a href="https://melodi-presto.mrcieu.ac.uk/">MELODI Presto</a> is a literature mining platform for mechanistic inference.</li>
  <li><a href="http://ldsc.broadinstitute.org/">LD Hub</a> supports online LD score regression analysis using a comprehensive manually curated database of GWAS studies.</li>
  <li><a href="http://melodi.biocompute.org.uk/">MELODI</a> is a literature mining platform to identify potential mechanistic pathways between exposures and disease outcomes.</li>
  <li><a href="http://fathmm.biocompute.org.uk/">FATHMM</a> predicts the functional effects of genetic variants in coding and non-coding parts of the genome.</li>
  <li><a href="http://mendelvar.mrcieu.ac.uk/">MendelVar</a> performs enrichment of ontology terms amongst genes linked to Mendelian disorders within genomic intervals.</li>
  <li><a href="http://www.mqtldb.org/">mQTLdb</a> is a database of methylation QTL (mQTL) from different stages across the lifecourse.</li>
  <li><a href="http://gtb.biocompute.org.uk/">Genome Tolerance Browser</a> visually compares functional prediction algorithms across the genome.</li>
  <li><a href="https://github.com/HAShihab/HIPred">HIPred</a> is an integrative approach to predicting haploinsufficient genes.</li>

</ul>
